src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/LinkLabel.LinkLabelAccessibleObjectTests.cs

```diff
@@ -73,5 +73,89 @@ public void LinkLabelAccessibleObject_Role_IsStaticText_ByDefault()
             Assert.Equal(AccessibleRole.StaticText, actual);
             Assert.False(linkLabel.IsHandleCreated);
         }
+
+        [WinFormsFact]
+        public void LinkLabelAccessibleObject_GetChildCount_IsExpected()
+        {
+            using LinkLabel linkLabel = new LinkLabel();
+
+            Random random = new Random();
+            int linkCount = random.Next(10);
+
+            for (int i = 0; i < linkCount; i++)
+            {
+                linkLabel.Links.Add(new Link(linkLabel));
+            }
+
+            int actual = linkLabel.AccessibilityObject.GetChildCount();
+
+            Assert.Equal(linkCount, actual);
+            Assert.False(linkLabel.IsHandleCreated);
+        }
+
+        public static IEnumerable<object[]> LinkLabelAccessibleObject_GetChild_TestData()
+        {
+            using LinkLabel linkLabel = new LinkLabel();
```

Comment:
Make like this everywhere
```suggestion
            using LinkLabel linkLabel = new();
```